baseline/OSPS-VM.yaml

```diff
@@ -94,4 +94,48 @@ criteria:
       status check passes before changes can be
       merged.
     control_mappings: # TODO
+
+  - id: OSPS-VM-05
+    maturity_level: 1
+    criterion: |
+      The project will publish contacts and process
+      for reporting vulnerabilities discovered within
+      the software code or dependencies.
+    rationale: |
+       # TODO 
+    details: |
+       Create a security.md file that contacts security 
+       contacts for the project and provide project's
+       vulnerability handling process.
+    control_mappings: # TODO    
+    security_insights_value: # TODO
+
+  - id: OSPS-VM-06
+    maturity_level: 2
+    criterion: |
+      The project MUST provide a means for reporting 
+      security vulnerabilities privately to the security
+      experts on within the project.
+    rationale: |
+      Security vulnerabilities should not be shared with 
+      the public until such time the project has been 
+      provide time to analyze and prepare remediations
+      to protect users of the project.
+    details: |
+       Enable private bug reporting through VCS or other
+       infrastrucuture.
+    control_mappings: # TODO    
+
+  - id: OSPS-VM-07
+    maturity_level: 2
+    criterion: |
+      The project MUST publish data about vulnerabilities
+      discovered within the codebase publicly.
+    rationale: |
+      Consumers of the project must be informed about
+      known vulnerabilities found within the project.
+    details: |
+       # TOTO
```

Comment:
We'll definitely want to provide more guidance about what this means. Do we mean projects must get a CVE? Does a post on social media count? What data needs to be published to satisfy this requirement?

Reply:
Issuing a cve is best case, but I wrote the requirement to account for things just as you state - a blog, a mailing list, a webpage.  the next level up we could add "issue a cve/become the cna for your work", but that's a future task to work through.  We already have guidance for projects on vuln mgmt and how to become a cna, we reference-wise we'er a good spot when this idea lands.

Reply:
Okay. I think this is reasonable, I just wanted to make sure I had a shared understanding.

Reply:
i've got a note, we have to go back and add/augment details and rationale.  Your concern will be explicitly spelled out in a details section

Reply:
I think we can address the confusion around -07 in the details as well